pkg/datapath/linux/node.go

```diff
@@ -1378,22 +1377,16 @@ func (n *linuxNodeHandler) NodeNeighDiscoveryEnabled() bool {
 
 // NodeNeighborRefresh is called to refresh node neighbor table.
 // This is currently triggered by controller neighbor-table-refresh
```

Comment:
It would be helpful to update the godocs for the new param. I find it difficult to understand why you would pass a refresh bool into a method that is called to refresh the node neighbor table.

Reply:
Good catch! Will update the docs to clarify.